### PR TITLE
Episode 13 

### PR DESCRIPTION
**Pull request by sockeqwe**
RSS Feed is valid: http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode13/feed.rss

TODO:

- [x] upload mp3 file. Please ensure that the file is correct (I think I have accidentally named it `The.Context.episode13.mp3.mp3`, obviously there should be only one `.mp3` at the end of the name)
- [ ] add git tag 
- [ ] update release section on github

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin) & Hannes Do
 
 ### [Episodes](https://github.com/artem-zinnatullin/TheContext-Podcast/releases)
 
+##### Episode 13: Conductor with Eric Kuck [Christian Bahl](https://twitter.com/EricKuck)
+  - [Show notes](https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_13.md)
+  - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/76)
+
```
Comment (ming13) [inline README.md]: Single empty line please.

File: feed.rss [hunk 2]
```diff
@@ -903,5 +903,65 @@
...
+			<ul>
+			<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/76">Discussion after the episode</a></li>
+			</ul>
+
+			<p><strong>Announcement:</strong> We have a new Co-Host for our podcast: Welcome <a href="https://twitter.com/arturdryomov">Artur Dryomov</a>!</p>
+
+			<p>We think that we need at least two hosts on each episode to produce a high quality content.
+			Unfortunately, producing podcasts is time consuming, but now that we are three hosts we can ensure that at least two of us have time to produce a new episode regularly.</p>
+
+			<p>In this episode Artur and Hannes chatted with Eric Kuck about <a href="https://github.com/bluelinelabs/Conductor">Conductor</a>, a framework created by Eric as alternative to Fragments. We discussed:</p>
```
Comment (ming13) [inline feed.rss]: `as _an_ alternative`?

File: show_notes/Episode_13.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/76)
+
+
+**Announcement:** We have a new Co-Host for our podcast: Welcome [Artur Dryomov](https://twitter.com/arturdryomov)!
+
+We think that we need at least two hosts on each episode to produce a high quality content.
+Unfortunately, producing podcasts is time consuming but now that we are three hosts we can ensure that at least two of us have time to produce a new episode regularly.
```
Comment (ming13) [inline show_notes/Episode_13.md]: `but now there are three hosts, so we can ensure`?

Comment (ming13) [inline show_notes/Episode_13.md]: `podcasts producing`?

File: show_notes/Episode_13.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Episode 13: Conductor with Eric Kuck
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/76)
+
+
+**Announcement:** We have a new Co-Host for our podcast: Welcome [Artur Dryomov](https://twitter.com/arturdryomov)!
+
+We think that we need at least two hosts on each episode to produce a high quality content.
```
Comment (ming13) [inline show_notes/Episode_13.md]: `high-quality`?

File: show_notes/Episode_13.md (hunk 3 above)
Comment (ming13) [inline show_notes/Episode_13.md]: `time-consuming`?

**Comment by artem-zinnatullin** [comment]
File uploaded!

File: show_notes/Episode_13.md (hunk 3 above)
Comment (sockeqwe) [inline show_notes/Episode_13.md]: podcast production?

Comment (sockeqwe) [inline show_notes/Episode_13.md]: 👍

File: show_notes/Episode_13.md (hunk 4 above)
Comment (sockeqwe) [inline show_notes/Episode_13.md]: 👍

File: show_notes/Episode_13.md (hunk 3 above)
Comment (sockeqwe) [inline show_notes/Episode_13.md]: 👍

File: README.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Before each episode we'll create an an [issue](https://github.com/artem-zinnatul
 
 ---
 
-Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin) & Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe)
+Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe) & [Artur Dryomov](https://twitter.com/arturdryomov).
```
Comment (artem-zinnatullin) [inline README.md]: Ha!

**Comment by artem-zinnatullin** [comment]
Sure, feel free to do that!

**Comment by artem-zinnatullin** [comment]
Or maybe you better wait for approve from @ming13 :)

**Comment by sockeqwe** [comment]
meh,  wanna publish this now :)

**Comment by sockeqwe** [comment]
Eric approved it

**Pull request merged**